src/prelude.rs

```diff
@@ -1,12 +1,15 @@
+#[doc(no_inline)]
```

Comment:
This makes the docs categorize these as re-exports.

Before, when searching the docs for "IronOxideErr", you'd get two results, `ironoxide::IronOxideErr` and `ironoxide::prelude::IronOxideErr`

Now it will only return `ironoxide::IronOxideErr` as intended

Reply:
That's awesome, I've been spending some time in the docs while doing the Swift stuff and noticed that weirdness. Thanks for adding it.